src/Reflection/Adapter/ReflectionProperty.php

```diff
@@ -47,15 +51,27 @@ public function getName()
      */
     public function getValue($object = null)
     {
-        throw new Exception\NotImplemented('Not implemented');
+        try {
+            return $this->betterReflectionProperty->getValue($object);
+        } catch (NoObjectProvided | NotAnObject $e) {
+            return null;
```

Comment:
I think this was fixed somewhen... 7.1 maybe?

Reply:
Nope, it throws a warning and returns `NULL`.

Reply:
Dang :-\